tests/test_version.py

```diff
@@ -1,5 +1,4 @@
-#!/usr/bin/env pytest -vs
-"""Version tests for packer project."""
+"""Version tests for pca-gophish-composition packer skeleton project."""
```

Comment:
Remove skeleton verbiage:
```suggestion
"""Version tests for pca-gophish-composition-packer project."""
```